OpenTabletDriver.UX/MainForm.cs

```diff
@@ -54,14 +54,12 @@ private Control ConstructPlaceholderControl()
             };
         }
 
-        public async Task Refresh()
+        public void Refresh()
```

Comment:
OutputMode combobox should be refreshed as well.

Reply:
Fixed